zeta.go

```diff
@@ -0,0 +1,12 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mathext
+
+import "github.com/gonum/mathext/internal/cephes"
+
+// Zeta computes the Riemann zeta function of two arguments
```

Comment:
Add a full-stop at the end of the sentence.

doc could also be a bit more detailed and/or point at:
- http://mathworld.wolfram.com/HurwitzZetaFunction.html
- https://en.wikipedia.org/wiki/Multiple_zeta_function#Two_parameters_case